services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/resource_tracker_core.py

```diff
@@ -99,22 +143,50 @@ async def _scrape_container_resource_usage(
         assert len(first_value) == 2  # nosec
         assert len(last_value) == 2  # nosec
 
-        container_resource_usage = ContainerResourceUsage(
+        user_id = int(metric["container_label_user_id"])
+        project_uuid = ProjectID(metric["container_label_study_id"])
+        node_uuid = NodeID(metric["container_label_uuid"])
+        user_email, project_info = await asyncio.gather(
+            *[
+                _get_user_email(osparc_repo, user_id),
+                _get_project_and_node_names(osparc_repo, project_uuid, node_uuid),
+            ]
+        )
+        project_name, node_label = project_info
+
+        service_key, service_version = metric["image"].split(":")
+        # Split the string at "/"
+        split_parts = service_key.split("/")
+        # Find the index of "/simcore/"
+        simcore_index = split_parts.index("simcore")
+        # Extract the desired part
+        service_key = "/".join(split_parts[simcore_index:])
+
+        container_resource_usage = ContainerScrapedResourceUsage(
```

Comment:
this is a pydantic model right? then you can use a validator instead of to_int_or_none. this will be cleaner.

Reply:
also I think we should remove totally the reservation unless should I put this in the labels as well?

Reply:
I can later in upcoming PRs clean not needed limits/reservations after we start to get the same and correct from dynamic/computational services  :+1: